remote_exec/remote_exec.proto

```diff
@@ -0,0 +1,59 @@
+syntax = "proto3";
+
+option go_package = "teleport_exec/remote_exec";
+package remote_exec;
+
+//-----------------------------------------------------------------------------
+message StartCommandRequest { string command = 1; }
+
+//-----------------------------------------------------------------------------
+message CommandStatusRequest { string command_id = 1; }
+
+message CommandStateResponse {
+  string command_id = 1;
+  string command = 2;
+  bool running = 3;
+  optional int32 result_code = 4;
+}
+
+//-----------------------------------------------------------------------------
+message StopCommandRequest { string command_id = 1; }
+
+message StopCommandResponse {
+  string command_id = 1;
+  bool success = 2;
+}
+
+//-----------------------------------------------------------------------------
+message CommandOutputRequest {
+  string command_id = 1;
+
+  // If the output should be streamed until the command stops
+  bool stream = 2;
+}
+
+message CommandOutputBlock { bytes output = 1; }
+
+//-----------------------------------------------------------------------------
+message StatusRequest {}
+
+message CommandStatus {
+  string command_id = 1;
+  string command = 2;
+  bool running = 3;
+}
+
+message StatusResponse {
+  string version = 1;
+  int64 pid = 2;
+  repeated CommandStatus commands = 3;
+}
+
+//-----------------------------------------------------------------------------
+service RemoteExec {
+  rpc Status(StatusRequest) returns (StatusResponse);
```

Comment:
Is `Status` for your own debugging? It's not required to implement, but if you are, I would re-use `CommandStateResponse` since this endpoint is missing things like `exit_code`.

Reply:
I was thinking it would be used for something like `docker ps` to list running commands + would provide an easy interface for debugging the state of the server in general. Good idea, I will reuse the `CommandStateResponse`.